utils/hexedit/Makefile

```
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2007-2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hexedit
+PKG_VERSION:=1.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/pixel/hexedit.git
+PKG_SOURCE_VERSION:=eab92dcaa34b66bc5182772afc9fda4ac8a27597
```

```suggestion
PKG_SOURCE_VERSION:=$(PKG_VERSION)
```

To checkout the tag directly.

---

Didn't know it could be done like that directly, thanks.